Biasing/src/Biasing/Utility/StepPrinter.cxx

```diff
@@ -9,43 +9,94 @@
 namespace biasing {
 namespace utility {
 
-StepPrinter::StepPrinter(const std::string& name, framework::config::Parameters& parameters)
+StepPrinter::StepPrinter(const std::string& name,
+                         framework::config::Parameters& parameters)
     : simcore::UserAction(name, parameters) {
   trackID_ = parameters.getParameter<int>("track_id");
+  processName_ = parameters.getParameter<std::string>("process_name");
+  depth_ = parameters.getParameter<int>("depth");
 }
 
-StepPrinter::~StepPrinter() {}
-
+bool StepPrinter::isDescendent(const G4Track* track) const {
```

Comment:
i vote for the spelling isDescendant, regardless of where this ends up being implemented

Reply:
Agreed with the spelling!

Reply:
Aren't both of those valid in this context? I'm by no means about to argue about this though, happy to change it

Reply:
I'm sure there's some semantics on why one over the other - I'm just used to seeing _descendant_, which is why I agreed with LK's comment. At the end of the day, not sure how much it matters, but maybe LK has a more robust argument that me (:

Reply:
my take was that descendant is a noun and descendent is an adjective. and from the discussion and culturally enforced old habits i suppose i was led to think of descendant on equal footing as ancestor (parent, daughter), and just thought it was a typo. but like you, einar, i won't argue about it 😄 i see your point now.